src/script/lua_api/l_settings.cpp

```diff
@@ -105,6 +106,24 @@ int LuaSettings::l_get_bool(lua_State* L)
 	return 1;
 }
 
+// get_np_group(self, key) -> value
+int LuaSettings::l_get_np_group(lua_State* L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	LuaSettings* o = checkobject(L, 1);
```

Comment:
Pointer misplaced: `LuaSettings *o`, same for `lua_State *L` in the function arguments.

Reply:
Will fix.